.github/workflows/main.yml

```diff
@@ -12,7 +12,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [jruby, 2.4, 2.5, 2.6, 2.7, 3.0, 3.1, truffleruby-head]
+        # Due to https://github.com/actions/runner/issues/849, we have to use
+        # quotes for '3.0' -- without quotes, CI sees '3' and runs the latest
+        # which is currently 3.1.1 as of 2022-02-28.
```

Comment:
Very nitpicky, but do we need this line `which is currently 3.1.1 as of 2022-02-28.`?

Reply:
Sure, I'm happy to remove that

Reply:
Updated!